### PR TITLE
Fix issues

### PR DESCRIPTION
**Pull request by anildigital**
Hi Nolan, thanks for awesome plugin.
I had to make these changes in order to use library correctly. Let me know if you have any suggestions if you see I am using it incorrectly. Thanks.

### CODE REVIEW

**Comment by ndarilek** [comment]
Thanks for your PR!

I'm a bit confused about why you changed the various plugin events to 
include the sender. My aim was to hide lower-level details, like the 
session and plugin IDs, from the client. The session and plugin pids 
should be enough of a unique identifier in cases where that is necessary.

In the case where you need to keep and associate state with a given 
session or plugin, I think GenEvent passes in the state along with the 
event, and accepts mutations in the reply.

I'm OK with some of these changes, like the ones where I obviously got 
the message format incorrect in plugin.ex. Would you mind splitting this 
into several distinct patches? Also, if I'm wrong about state and you 
*do* in fact need the sender ID then please let me know, but in general 
I think that should be handled when the handler is attached to the 
plugin by creating an empty piece of state.

Thanks again.

File: lib/janus/session.ex [hunk 1]
```diff
@@ -124,19 +128,22 @@ defmodule Janus.Session do
           event_manager = session.event_manager
           case data do
             %{janus: "keepalive"} -> GenEvent.notify(event_manager, {:keepalive})
+            %{janus: "event", plugindata: plugindata, sender: sender} ->
+              jsep = data[:jsep]
+              GenEvent.notify(event_manager, {:event, pid, data, jsep, sender})
```
Comment (ndarilek) [inline lib/janus/session.ex]: This is still including the sender. Is there any reason you need the explicit sender and can't match against `pid`, which would be unique per sender anyway?

File: lib/janus/session.ex [hunk 2]
```diff
@@ -87,7 +87,11 @@ defmodule Janus.Session do
   def destroy(pid) do
     base_url = Agent.get(pid, &(&1.base_url))
     plugin_pids = Agent.get(pid, &(&1.handles)) |> Map.values()
-    plugin_pids.each(&(Janus.Plugin.detach(&1)))
+    IO.puts "See plugin pids"
+    IO.inspect plugin_pids
+    Enum.each (plugin_pids), fn (pid) ->
```
Comment (ndarilek) [inline lib/janus/session.ex:90]: These `inspect` calls should probably go.

**Comment by anildigital** [comment]
Hi @ndarilek, I have done changes as per your code review. Could you please review it again? Let me know if you want further more changes. Thanks.

**Comment by ndarilek** [comment]
Thanks!

**Pull request merged**